src/components/Action/Action.vue

```diff
@@ -69,6 +69,10 @@ export default {
 					}
 				]
 			}
+		},
+		forceWhite: {
```

Comment:
I'm guessing that since we moved to the font icons, we can now set it on the parent component, so we might not need the props now :thinking: